python/paddle/hapi/model_summary.py

```diff
@@ -77,7 +78,8 @@ def forward(self, inputs):
 
             lenet = LeNet()
 
-            params_info = paddle.summary(lenet, (1, 1, 28, 28))
+            input_data = [paddle.ones([1, 1, 28, 28])]
```

Comment:
do not delete the original examples. Add new examples at the back, and add a example of dict and list input.

Reply:
done